hadeoh/WayFarer
# #167123362 Implement user book seat endpoint

#### What does this PR do?

- Implement user should be able to book a seat on a trip 

#### Description of the tasks to be completed

- create a method for booking a seat in the bookings model

- create a method for user to book a seat in the bookings service

- create a method for user to book a seat in the bookings controller

- create a route for user to book a seat in the bookings routes

- create validations for user to book a seat in the middleware

- write unit tests for the booking endpoint

#### How should this be manually tested?

- Clone the repository using git clone https://github.com/hadeoh/WayFarer.git . in your terminal

- run npm install on your terminal

- run npm test to see test results

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories

- PT Story link - [#167123362] (https://www.pivotaltracker.com/story/show/167123362)

#### Screenshots (if appropriate)

- N/A

#### Questions

- N/A
